services/sidecar/src/simcore_service_sidecar/executor.py

```diff
@@ -49,6 +50,15 @@ def create(self) -> None:
                 shutil.rmtree(str(folder))
             folder.mkdir(parents=True, exist_ok=True)
 
+    def delete(self) -> None:
+        for folder in [
+            self.input_folder,
+            self.output_folder,
+            self.log_folder,
+        ]:
+            if folder.exists():
+                shutil.rmtree(str(folder))
```

Comment:
``rmtree`` here can [raise exception](https://docs.python.org/3/library/shutil.html#shutil.rmtree

I would suggest a "soft delete" ... worst case, there is a "stale" folder in the host that can be later pruned
```python
  def log_error(function, path, excinfo):
     log.warning( "Failed to remote %s [%s]. Should consider pruning files in host later", path, excinfo)

  shutil.rtree(str(folder), onerror=log_error )
```

Reply:
great!